Source/control.cpp

```diff
@@ -606,7 +606,7 @@ void __cdecl DrawSpellList()
 				v17 = 636;
 			}
 LABEL_68:
-			v20 += 56;
+			v20 += 14;

```

Comment:
Good catch, since the size of the pointer element is 4 now and now 1, each increment to the pointer is multiplied by its element size.